src/diem/testing/miniwallet/config.py

```diff
@@ -31,10 +43,11 @@ def account(self) -> LocalAccount:
 
     @property
     def server_url(self) -> str:
-        return "%s://%s:%s" % (self.url_scheme, self.host, self.port)
+        return str(self.server_conf)
```

Comment:
from what I can understand, it is really just:
```
    if. self.base_url: 
           return self.base_url
    return "%s://%s:%s" % (self.url_scheme, self.host, self.port)
```
adding ServerConfig and __str__ and post init behaviors makes it more confusing.